server/src/main/java/io/spine/server/event/EventBus.java

```diff
@@ -107,6 +108,9 @@
     /** The validator for messages of posted events. */
     private final MessageValidator eventMessageValidator;
 
+    /** The handler for dead events. */
+    private final DeadMessageTap<EventEnvelope> deadMessageHandler;
```

Comment:
Why we have `handler` in the field name, and `Tap` in the type?

Can we hide the implementation details, and have more generic type for the field?